docs/docs/client.html

```diff
@@ -64,10 +64,12 @@
   def genView(value: Boolean): CalibanSettings                  // Provide a case class and helper method to select all fields on an object (default: false)
   def scalarMapping(mapping: (String,String)*): CalibanSettings // A mapping from GraphQL scalar types to JVM types, as unknown scalar types are represented as String by default.
   def imports(values: String*): CalibanSettings                 // A list of imports to be added to the top of a generated client
+  def splitFiles(value: Boolean): CalibanSettings               // Split single client object into multiple files (default: false)
```

Comment:
Ah, to modify the docs it's not directly in the html, look at the .md files in the `vuepress` folder.

Reply:
Fixed, thanks